src/aiperf/metrics/types/spec_decode_metrics.py

```
@@ -0,0 +1,317 @@
+    tag = "spec_decode_token_weighted_acceptance_length"
+    header = "Token-Weighted Acceptance Length"
+    short_header = "Token-Wtd Accept Len"
+    short_header_hide_unit = True
+    unit = GenericMetricUnit.RATIO
+    flags = MetricFlags.LARGER_IS_BETTER
+    console_group = MetricConsoleGroup.SPEC_DECODE
+    display_order = 5010
+    required_metrics = {
+        TotalAcceptedDraftTokensMetric.tag,
+        TotalSpecDecodeStepsMetric.tag,
+    }
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)